zcash_client_backend/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 //! light clients.
 
 // Catch documentation errors caused by code changes.
-#![deny(intra_doc_link_resolution_failure)]
+#![deny(broken_intra_doc_links)]
```

Comment:
I don't know whether we want to make this change yet (also in the other crates). It's annoying that it results in a lot of false-positive annotation warnings.